tests/test_tokenization_wav2vec2.py

```diff
@@ -0,0 +1,336 @@
+#!/usr/bin/env python3
+# coding=utf-8
+# Copyright 2021 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for the Wav2Vec2 tokenizer."""
+import inspect
+import json
+import os
+import random
+import shutil
+import tempfile
+import unittest
+
+import numpy as np
+
+from transformers.models.wav2vec2.tokenization_wav2vec2 import VOCAB_FILES_NAMES, Wav2Vec2Tokenizer
+
+
+global_rng = random.Random()
+
+
+def floats_list(shape, scale=1.0, rng=None, name=None):
+    """Creates a random float32 tensor"""
+    if rng is None:
+        rng = global_rng
+
+    values = []
+    for batch_idx in range(shape[0]):
+        values.append([])
+        for _ in range(shape[1]):
+            values[-1].append(rng.random() * scale)
+
+    return values
+
+
+class Wav2Vec2TokenizerTest(unittest.TestCase):
+    tokenizer_class = Wav2Vec2Tokenizer
+
+    def setUp(self):
+        super().setUp()
+
+        vocab = [
+            "<pad>",
+            "<s>",
+            "</s>",
+            "<unk>",
+            "|",
+            "E",
+            "T",
+            "A",
+            "O",
+            "N",
+            "I",
+            "H",
+            "S",
+            "R",
+            "D",
+            "L",
+            "U",
+            "M",
+            "W",
+            "C",
+            "F",
+            "G",
+            "Y",
+            "P",
+            "B",
+            "V",
+            "K",
+            "'",
+            "X",
+            "J",
+            "Q",
+            "Z",
+        ]
```

Comment:
Exceptionally, and to make the rest more readable, I think we can add the marker to ignore styling here and have all of this on a very long line. :-)